### PR TITLE
Add Questions? section to README

### PR DESCRIPTION
**Pull request by bbuchalter**
To address https://github.com/solidusio/solidus/issues/1318

See https://github.com/bbuchalter/solidus/tree/add-support-to-readme#quesitons for visual review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,9 @@ and the admin can be found at [http://localhost:3000/admin/](http://localhost:30
 
 As part of running the above installation steps, you will be asked to set an admin email/password combination. The default values are `admin@example.com` and `test123`, respectively.
 
+### Quesitons?
```
Comment (tvdeyen) [inline README.md]: Typo. It has to be "Questions?" ;)

**Comment by tvdeyen** [comment]
:+1:

**Comment by jhawthorn** [comment]
:+1:

**Comment by bbuchalter** [comment]
Typo is fixed. ;)

**Pull request merged**